### ISSUE TITLE
Allatini chapter : Schola, Cristiana, Kat, Eleni

### ISSUE THREAD

**Issue by RJP43** [RJP43/LiliElbe_EngagedLearners#31]
hey there @schola0208, @bcristiana, @kklimek1, and @elenipamboukis 👋  
  
Your team has been assigned the French-language Allatini chapter. As a team, you will create one file containing both the already-encoded translation and the to-be-encoded original transcription of this text. This text is rather unique so be sure to reach out via this issue if you hit any snags. Simply tag me - @RJP43 - and add the "help wanted" label to this issue.  
![screenshot highlighting help wanted label](https://user-images.githubusercontent.com/8645521/54369274-a2d2fc00-4643-11e9-8a83-c19196fce6bd.png)  If I do not respond within 48 hours with my 👀 reaction then please shoot me an email at `rparker3 (at) luc (dot) edu` and I will be in contact ASAP. 
  
Two of you should begin by associating [the new project schema](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/LEDA_ODD.rng) to the encoded-translation XML file:  [AllatiniChapter.xml](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/archivalMaterials/French_archive/HAllatiniChapter/AllatiniChapter.xml). Then you will need to correct the XML errors (in the entire file - including errors found in the `<tei:header>` elements). After you have uploaded the valid and well-formed XML file, tag me - @RJP43 - and add the "transformation ready" label to this issue.  I will then use our project's XSLT (eXtensible Stylesheet Language Transformation) and will respond to your team here in this issue with an HTML proofing link. From there your team will work to [HTML proof](https://github.com/RJP43/LiliElbe_EngagedLearners/wiki/Proofing-HTML) the HTML rendition of this text's translation -- reporting errors and design concerns regarding the HTML styling to me via this issue (tagging me and using the "help wanted" label) and, otherwise, revising the XML where improper encoding has resulted in faulty HTML rendering.  As a reminder, you will use all of the materials in the [Allatini Chapter folder](https://github.com/RJP43/LiliElbe_EngagedLearners/tree/master/ProjectDocs/archivalMaterials/French_archive/HAllatiniChapter) (i.e. original transcriptions, facsimile images, etc.) to proof the HTML.  

The other two of you will work on transcribing and encoding (directly in oXygen) the original French of this text using the [facsimile images](https://github.com/RJP43/LiliElbe_EngagedLearners/tree/master/ProjectDocs/archivalMaterials/French_archive/HAllatiniChapter/ChapterImages). Once encoding is complete on the original French your team will consult [our project's encoding guidelines](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/encodingGuidelines.md) and combine the two files into a single file named `HAllatiniChapter_combo.xml`. Then @ProfPLC and I will inform your project team of next tasks or that your project contribution (well... at least this more "technical" aspect of it) is complete.  Please note the project and the training materials available on this repository are still in flux. This means that there might be moments where I need to inform your team of new updates that will require XML corrections or re-reading of lesson materials; therefore, please check this issue and the general [issues board](https://github.com/RJP43/LiliElbe_EngagedLearners/issues) regularly for alerts from me.   

**_Recent Update_**: We have moved away from using the MIWSchema.rng, and are now using our new schema - [LEDA_ODD.rng](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/LEDA_ODD.rng).  Please be sure you download this new schema and delete the old one from your computers.  We also now have a much more robust prosopography file listing all of the identified people and places in our project files. Please reference our [LEDA_Prosopography.xml](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/LEDA_Prosopography.xml) when trying to identify proper key attribute values.  Every xml:id attribute value in the file is a possible key attribute value that can be used in your XML files. We will be outputting an HTML transformation of this prosopography file soon and when we do I will be sure to inform your team via this issue.

**Comment by kklimek1** [comment]
I'd be okay with doing the second task, with the encoding and transcribing!

**Comment by bcristiana** [comment]
I find interesting both tasks, but I would rather join @kklimek1 in transcribing and encoding--it would be a great way to refresh my French. Whenever you will see this, let me know, @kklimek1 , @schola0208  and @elenipamboukis, if is OK with you.

**Comment by RJP43** [comment]
:eyes: @bcristiana be careful not to close your team's issue when commenting. Please click the green comment button to post your responses. Thanks :+1:

**Comment by RJP43** [comment]
❗️ ❣️  hey team  @schola0208, @bcristiana, @kklimek1, @elenipamboukis

Just popping in to let you know @ProfPLC had me add Anne's translation to your project team's folder. Here is the quick link to the PDF - [H l ne Allatini Il et Elle He and She from Mosa ques 1939.pdf](https://github.com/RJP43/LiliElbe_EngagedLearners/blob/643d7b289651c87a48e89f3cfd7c9b5b2b716cb2/ProjectDocs/archivalMaterials/French_archive/HAllatiniChapter/H%20l%20ne%20Allatini%20Il%20et%20Elle%20He%20and%20She%20from%20Mosa%20ques%201939.pdf)

**Comment by elenipamboukis** [comment]
@bcristiana that is okay with me! Hey team, I would prefer to work on the first task!

**Comment by schola0208** [comment]
@bcristiana @elenipamboukis @kklimek1 I'm happy either way. I would do whatever task is left on the list

**Comment by ProfPLC** [comment]
Dear Allatini team, 
The translation by Dr. Anne Callahan that Rebecca sent you in a previous comment is a revision of the translation that was encoded last semester.  So those on task 1 will need to revise the translation, re-encoding where necessary before associating the schema.  This is work you must do first. Rebecca and I can meet with you to go over this task.

**Comment by elenipamboukis** [comment]
Class March 19-
@schola0208 and I are doing assignment 1, we will meet together and separate our work
@bcristiana and @kklimek1 are doing assignment 2, Christiana is doing first half of pictures and Kat is doing that second half

**Comment by elenipamboukis** [comment]
@RJP43 Schola and I found that this error: `key="madrid">Madrid</placeName>` is not valid. We looked Madrid up in the prosopography and the key does not exist as a location name.

**Comment by RJP43** [comment]
@elenipamboukis I will add it to the prosopography. Thanks! 👍  Please leave the encoding in the XML file and when the next schema release goes out in a few days the error will disappear.

**Comment by elenipamboukis** [comment]
Today March 21 from 11:00-1:00, Eleni & Schola: Meet together updating translation by reading aloud together and comparing it, From line 1-500 in the XML and page 124-142 in the updated translation.

**Comment by kklimek1** [comment]
@RJP43 Hi Rebecca! I'm attempting to encode my half of the pictures, but I've hit a snag. The pages all have "Mosaiques" at the top of the page, and I'm not sure how to incorportate that into the XML file. Would I be encoding that as `<pb style="heading" rend="___"/>` ? Thanks!

**Comment by RJP43** [comment]
:eyes: @kklimek1 Huzzah, you are correct. 🎉 Well done finding the correct encoding in the guidelines. Now for the `rend` attribute value paste the exact transcription/translation of that heading text. So your encoding for those should look like this:  
`<pb style="heading" rend="Mosaïques"/>`  👍

**Comment by bcristiana** [comment]
@RJP43 , good evening! I tried to send you my two `Mozaiques`, 10 and 11, and wonder if you've been able to open them. I don't know how otherwise to show you my work without uploading it. Do you have any time tomorrow, after 12.30? You could see what I've done and tell me if I am on the correct path.

**Comment by RJP43** [comment]
:eyes: @bcristiana You will need to upload your materials to GitHub not only so I can see your progress but so your teammates can too. We can schedule a meeting for Monday or Tuesday to go over how to upload and to check your progress if you like. Unfortunately, I do not have any availability today to meet. 📆 For Monday, I am open Monday before 11 AM or between noon and 2 PM. For Tuesday, I can meet anytime before the Engaged Learning class at 2:30 PM.  Please let me know when works best for you. 💟

**Comment by kklimek1** [comment]
@RJP43 
Would you mind meeting with me around 1:00 on Tuesday? Similarly to @bcristiana I want to make sure that I'm doing everything correctly!

**Comment by RJP43** [comment]
:eyes: Sure thing @kklimek1! I have you on [my schedule](https://calendar.google.com/calendar/embed?src=a72eb2hjpuf5g45dpdo8t1oer0%40group.calendar.google.com&ctz=America%2FChicago) for 1 PM on Tuesday (3/26) in IC Study Room 217.

**Comment by bcristiana** [comment]
@RJP43 , could you meet me Monday? My class starts at 11 am.

**Comment by RJP43** [comment]
:eyes: @bcristiana Monday is fine. Do you want to meet before your 11 AM  class or after? If after, what time in my noon to two slot works best for you?

**Comment by bcristiana** [comment]
@RJP43 , thank you.  The class ends at 12.40, so I could come see you whenever it is more convenient for you: either before 11 am or after 12.40 pm.

**Comment by bcristiana** [comment]
@RJP43 , if Monday is still fine, where do I meet you and when?

**Comment by RJP43** [comment]
:eyes: @bcristiana let's meet at 12:45ish in IC Study Room 214. 📆 I have added you to [my office hours schedule](https://calendar.google.com/event?action=TEMPLATE&tmeid=NmVsYW5vOTRpM2k3cWFycW43OHBqdG1vcDQgYTcyZWIyaGpwdWY1ZzQ1ZHBkbzh0MW9lcjBAZw&tmsrc=a72eb2hjpuf5g45dpdo8t1oer0%40group.calendar.google.com). I have reserved 214 for the majority of the afternoon if others also need to meet.

**Comment by bcristiana** [comment]
@RJP43 , thank you. I will be seeing you Monday.

**Comment by elenipamboukis** [comment]
@RJP43 Schola and I just finished comparing the two translations and fixing what was missing/ incorrect in the XML. We found some things that didn't necessarily make sense when we were correcting the XML. For example, the XML on line 833 said, "like all or almost all operations," which made sense to us rather than the corrected text which says on page 137, "like almost all or almost all operations." We are just a bit confused, because that wasn't the first example of something that didn't make a lot of sense. I have a list of notes on my phone of other examples. 

Also- are we still waiting on the new schema to move onto the next task? The next task for us would be to correct the XML errors.

**Comment by RJP43** [comment]
:eyes: @elenipamboukis please update your file by uploading your latest XML to your team's folder over top of the old XML so I can better understand where you are in the process of completing your team's tasks. I have recently updated the prosop. and schema to include the missing Madrid key and the most recent file in your folder suggests you have already associated the new schema to the XML prior to making the corrections.

**Comment by bcristiana** [comment]
@RJP43  and team, I gladly let you know that I uploaded my contribution to the task. There are still two pages--the title page and the cover of the memoir--that I didn't upload: the first, because it has to precede one of @kklimek1 's pages, and the latter, because I need suggestions about how to encode it.

**Comment by bcristiana** [comment]
@RJP43 , thank you for your help! I was so relieved to finish this part of the project, that I forgot to state the obvious: your help. So thank you again!

**Comment by kklimek1** [comment]
@RJP43 Thank you for meeting with me! As I was encoding, I stumbled upon "Marcus Hierschfeld" in the text, but in the prosopography there was only `<persName>Magnus Hirschfeld</persName>`.

**Comment by RJP43** [comment]
:eyes: @kklimek1 This is one where I wonder if this is a spelling error on Allatini's part. Is there any information in the surrounding text indicating Marcus is of relation to Magnus or does it seem Allatini is actually talking about Magnus? Might be worthwhile to pose the question to @ProfPLC before/after class: Does Magnus have a brother Marcus? Is Marcus an alternate name for Magnus?  

Please report back. If it is a spelling error on Allatini's part I can give you the encoding to indicate this, but if it turns out this is a separate person all together I will add him to the prosopography. 👍 Thanks!

**Comment by RJP43** [comment]
:eyes: @kklimek1 @ProfPLC Also we will want to take note of the different spellings of the last name. Is this just how the French would write Hirschfeld or is this an even bigger indicator that Allatini botched Magnus Hirschfeld's name?

**Comment by ProfPLC** [comment]
Marcus is a mistake, this is Magnus Hirschfeld and should be encoded as such. Do you know if Marcus is in the original? I believe it's Allatini's mistake but we want to make sure it's not the translator's.

**Comment by elenipamboukis** [comment]
@RJP43 Schola and I have updated our XML document with your new schema, and have corrected all errors in the document besides the Frauenkilink place key because we do not know which of they four keys in the pros. to use for it.

**Comment by kklimek1** [comment]
Today in class @bcristiana and I realized that there was a little bit of a miscommunication. We ended up having some overlap in the facsimile images that we transcribed and encoded. So while I finish up the images 28 through 41, Cristiana will finish images 5 through 9. Once we both finish up, I'll be taking both of our XML documents and turning them into one file.

**Comment by schola0208** [comment]
> @RJP43 Schola and I have updated our XML document with your new schema, and have corrected all errors in the document besides the Frauenkilink place key because we do not know which of they four keys in the pros. to use for it.

@RJP43 , we figured out the Frauenkilink key, with the help of @ProfPLC , and we figured out the translation issues with her as well. @elenipamboukis is uploading the new corrected xml file. I'm adding the transformation ready tag

**Comment by schola0208** [comment]
the transformation ready tag applies to only @elenipamboukis and @schola0208

**Comment by kklimek1** [comment]
@RJP43 I'm still a little confused on how I do the persName element around certain people in the document. For example Professor W., in the prosopography, this is what I'm looking at: `  <person xml:id="warnekros" source="historical" synch="#kreutz">`
               `<persName>Kurt Warnekros</persName>`
               `<persName>Warnekros</persName>`
               `<persName>Professor Warnekros</persName>`
               `<persName>Professor W</persName>`
               `<persName source="#frenchSupp">Professeur W</persName>`
               `<note/>`
            `</person>`
Do I use this whole fragement? Or do I just put a `<persName>` element without any `key=`? Thank you!

**Comment by kklimek1** [comment]
> @RJP43 I'm still a little confused on how I do the persName element around certain people in the document. For example Professor W., in the prosopography, this is what I'm looking at: ` <person xml:id="warnekros" source="historical" synch="#kreutz">`
> `<persName>Kurt Warnekros</persName>`
> `<persName>Warnekros</persName>`
> `<persName>Professor Warnekros</persName>`
> `<persName>Professor W</persName>`
> `<persName source="#frenchSupp">Professeur W</persName>`
> `<note/>`
> `</person>`
> Do I use this whole fragement? Or do I just put a `<persName>` element without any `key=`? Thank you!

Actually I think I figured it out, would it look like this: `<persName key="warnekros">professeur W.<persName/>`?

**Comment by RJP43** [comment]
:eyes: @kklimek1 you got it!

**Comment by bcristiana** [comment]
Hi, @kklimek1, team and @RJP43 , I finished encoding the facsimiles 5-9 and added them to the final Mozaiques_bc.xml, as well as the comments about facsimiles 1 and 2. Now I need to remember how to upload the final version of the Mozaiques_bc.xml.

**Comment by bcristiana** [comment]
@RJP43 , I tried to remind you about Lili's French IDs and I am certain that I have attached the `help wanted` label to the wrong thing, so that there is a `new issue` assigned to you and, somehow, to myself (had I known how to attach emojis, I would have put something here).

**Comment by RJP43** [comment]
@bcristiana 👀 🎉 thank you so much for creating a new issue for the couple of specialized items you need help with I will go ahead and answer you on that issue - #37. Seriously, well done with the label and everything - _that is a perfect example of what to do_!  💟 

As for getting your file into your team folder (so @kklimek1 can reference it and the two of you can move forward with combining your two encoding styles) I have just tagged the class in a comment about proper GitHub workflow (including upload/download habits) on issue #35. I hope that walk-through helps. Please let us know if you succeed in uploading so we can celebrate :tada: OR let us know if you still need help sharing your file so we can  📆 schedule a meeting during my office hours or have you meet with your partner(s) that are confident in their upload abilities before/after/in class. 🤝  

Though by the looks of your commit history - https://github.com/RJP43/LiliElbe_EngagedLearners/commits?author=bcristiana - it seems you were able to upload - 🎉 ❓

**Comment by bcristiana** [comment]
@kklimek1 , hi! Just wanted to let you know that I uploaded the revised version of Mozaiques_bc.xml, this time with new info added to the TEI header, as provided by the TEI header spreadsheet. Please let me know if you find it, because I cannot see it, though the uploading seems to have been successful. It is the file that contains the title page and pages 201-223, as well as the comments about the physical condition of each facsimile. @RJP43 , thank you for the explanations, I tried to follow them as best as I could. We will see if Katy can retrieve it. Honestly, I can't wait to share into whatever the celebratory emoji is about.

**Comment by kklimek1** [comment]
Sorry for the late comment, but the combined document has been uploaded (and then reuploaded because I missed a little part of the text) !

**Comment by RJP43** [comment]
Hey there @schola0208, @bcristiana, @kklimek1, and @elenipamboukis 👋 I have reserved IC Study Room 217 for our 10 AM meeting on Friday. See you all there!

**Comment by elenipamboukis** [comment]
Schola will update the complete translation file and Cristiana will update the TEI header

**Comment by elenipamboukis** [comment]
After meeting with Rebecca at 10:00 in a study room, @kklimek1 will be uploading the new combination file onto the Allatini Chapter file, and @bcristiana will update the new TEI header.

**Comment by elenipamboukis** [comment]
@kklimek1 will be doing place name attributes and I will be doing person name attributes both of us making sure the French matches the English attributes. 
@bcristiana and @schola0208 will be doing the segments in the paragraphs.

**Comment by kklimek1** [comment]
@RJP43 Should we be putting persName elements around pronouns and nouns that indirectly refer to the person? So if Einar is referred to as "my fiance"?

**Comment by ProfPLC** [comment]
:eyes: @kklimek1 and @elenipamboukis -- Pronouns do not get persName elements. If you can identify the reference, such as "my fiance," from context, then put the persName element in with the appropriate key ID. But if you're not sure of the reference, run these by me via email (pcaughi@luc.edu). For example, a reference to my fiance by Gerda could be Einar but also Fernando Porta. I could help identify which one.

**Comment by bcristiana** [comment]
@RJP43 , hi, this is a message from both me and @schola0208 . This is the problem we ran into: the french text has paragraph 4 and 5 separate, while the translation has them combined into one. We thought to use different segment numbers to mark the differences btw paragraphs, but that would mean we have two segments in one paragraph in the english transaltion. @schola0208 is pasting what it looks like in the next comment. So is it ok to have two segments in one paragraph, or is there another way to segment it when paragraph divisions don't match? 

 `<p><seg n="f04">Nous partîmes le même soir en voyage de noces, pour <placeName key="rome">Rome</placeName>.</seg></p>
                <p><seg n="f05">Dès mon retour, cinq mois plus tard, nos relations amicales reprirent, de plus en plus cordiales et affectueues. Je venais de perdre ma mère et j'attendais mon premier enfant? Mes amis scandinaves m'entourèment et je trouvai un grand réconfort dans leur affection les jours où, sans nouvelles de mon mari, ils se multiplaient en attentions et tâchaient de me divertie de leur mieux, pour me détoruner de mon angoisse.</seg></p>`

**Comment by schola0208** [comment]
`<p><seg corresp="#fo4">We left that same evening for our honeymoon in <placeName key="rome"
                        >Rome</placeName></seg>. <seg corresp="#f05">Upon my return, five months later, we took up again our
                    friendly relations that became more and more cordial and affectionate. I had
                    just lost <persName>my mother</persName> and was
                    expecting <persName key="allatiniJ">my first child</persName>. My Scandinavian
                    friends embraced me and I found great comfort in their affection on days when,
                    without news of my <persName key="allatiniE">husband</persName>, they increased
                    their attention and tried to distract me as best as they could, to turn my thoughts
                    away from my anxiety.</seg></p>`

**Comment by RJP43** [comment]
:eyes: @schola0208 @bcristiana I think this is an instance where you need to instead divided the translation and original paragraphing so they match the original print rendition. If there are two paragraphs in the facsimile images then both the translation and the original French need to be divided according to the print. I believe that will also solve the `<seg>` issue but please comment back and let me know if not. Thanks 👍

**Comment by bcristiana** [comment]
@RJP43 , thank you for the explanation. Now we know that the french text is the one to which we compare the translation. When--not if--we meet other obstacles, we'll ask for your guidance..

**Comment by bcristiana** [comment]
@kklimek1 , @elenipamboukis , @schola0208 , I have uploaded on GitHub the HAllatiniChapter_combo.xml. It contains the french text completely segmented. @schola0208 will add soon, in a couple of hours, the segmented translation, so maybe you should wait for both texts. I also have to rework the TEI header, but I hope I can do it from the html version.

**Comment by bcristiana** [comment]
@kklimek1 , @elenipamboukis , @schola0208 , hi again! I have uploaded on GitHub a second file, 1HAllatiniChapter_combo.xml, with the newly edited TEI header, so please work on/with this version.

**Comment by schola0208** [comment]
@kklimek1 @elenipamboukis @bcristiana I have uplaoded the file with what me and @bcristiana were able  to do. here is the link https://github.com/RJP43/LiliElbe_EngagedLearners/blob/master/ProjectDocs/archivalMaterials/French_archive/HAllatiniChapter/HAllatiniChapter_combo.xml. Pay attention because there is another file by the same name but that starts with a 1; that file is not valid but I couldn't figure out how to delete it. Also when you start working on this file post on the issues board so that no one else works on it, and let us know how long you intend to work on it so that others can budget their time accordingly

**Comment by elenipamboukis** [comment]
@RJP43 Hello Rebecca! Can you let me know how to search up person names again for the whole XML? Thank you!

**Comment by RJP43** [comment]
:eyes: My apologies @elenipamboukis! If you go to the far right corner you will see the XPath window.  
   
![screenshot with red square highlighting the XPath Window in oXygen](https://user-images.githubusercontent.com/8645521/55976242-52a28600-5c51-11e9-991b-7be2dae4d3fe.png)  
  
This window is kind of like the find feature (Ctrl/Command F) except it searches on the XML's hierarchy. So for example to get a sense of how many people are marked across the entire file you would run the following XPath:  `//persName`.  The `//` tells oXygen to go down from the root element (`<TEI>`) and look through all the descendant elements and then the `persName` indicates the element you specifically want to examine. When you hit enter after typing in the XPath your results will appear at the bottom of your screen and all of that particular element will be highlighted in the main text window.  
  
![screenshot with red squares highlighting the XPath Results in the bottom of the oXygen window](https://user-images.githubusercontent.com/8645521/55976542-32bf9200-5c52-11e9-9240-2079766a48b5.png)  
  
You can edit the element you want to see by changing the XPath. Here are the few I showed you in class on Tuesday:  
-  To find all of the people marked in `<persName>` in the original:  `//div[@type="original"]//persName`
-  To find all of the people marked in `<persName>` in the translation:  `//div[@type="translation"]//persName`
-  To find all of the places marked in `<placeName>` in the original: `//div[@type="original"]//placeName`  
-  To find all of the places marked in `<placeName>` in the translation: `//div[@type="translation"]//placeName`  

And here are some extra ones that might be useful: 
-  To find all of the people marked in `<persName>` in the original but missing a key ID: `//div[@type="original"]//persName[not(@key)]`  
-  To find all of the people marked in `<persName>` in the original that have a key ID: `//div[@type="original"]//persName[@key]`  
- To find a count of the paragraphs in original:  `count(//div[@type="original"]//p)`  

If it interests you or any of your teammates (@kklimek1 @bcristiana @schola0208) to learn more about the magic you can do with XPath here are a couple of my favorite resources:   
- [Elisa Beshero-Bondar's "Follow the XPath!"](https://newtfire.org/courses/dh/explainXPath.html) 
- [David J. Birnbaum's "The XPath functions we use most"](http://dh.obdurodon.org/functions.xhtml)

**Comment by schola0208** [comment]
@bcristiana @kklimek1 @elenipamboukis I am going to be working on the file till 1pm,  I will re-upload it then.

**Comment by schola0208** [comment]
> @bcristiana @kklimek1 @elenipamboukis I am going to be working on the file till 1pm, I will re-upload it then.

I am done with my session, I have re-uplaoded it under the original name

**Comment by elenipamboukis** [comment]
@RJP43 Hello! I am having GREAT trouble with persName. I feel like I have been working on trying to add more names within the translation for ages and I am not really seeing any progress as the number of persName attributes goes up so slowly. I do not want it to seem as though I am cheating my hours for how long I am working on it, but I can't seem to wrap my brain around what I am doing effectively!

**Comment by elenipamboukis** [comment]
@RJP43 In following up with my recent comment `//div[@type="original"]//persName[not(@key)]` helps, but most of the missing 'key types' Christiana has noted, but those names are not in the posopography

**Comment by elenipamboukis** [comment]
@RJP43 in working on the persName attributes, I am trying my best to look at the prosopography and add key ID's; however some key ID's are missing and they are noted in comments within the XML. The original has 52 total issues regarding missing key ID's to persNames, and the translation has 10 total issues regarding missing key ID's to persName. In total after trying to help even the number difference of 337 persNames in the original and now 278 persNames in the translation, I think that by you adding the missing ones to the prosopography and looking at the comments attached to the missing key ID persnames can help level the number of persNames for the combo document. I am trying my best to level out the number, but it is incredibly hard! I also think that some of the persName in the original document do not need to be noted. Though, Christiana's points are valid. I have uploaded the most recent combo XML if you would like to look at it! Thank you!

**Comment by schola0208** [comment]
@kklimek1 @bcristiana @elenipamboukis I am working on the file for the next 20 minutes or so. I'll notify you when I upload it

**Comment by schola0208** [comment]
@RJP43, seg 72 and seg 73 are two distinct paragraphs in the facsimile, but their seg corresp in the translation is one paragraph that doesn't adhere to the order of sentences or words. We had talked in class about doing one seg for the two paragraphs, but XML editor doesn't let me do that because seg has to be inside paragraphs markers. So I estimated and separated the one paragraph into two so that segs can match their seg corresps. 
Also, I left the last paragraph that says'translated by.....' without its own segment because that paragraph is not in the original so it wouldn't have any corresp. 
If this is ok, then the file is transformation ready as far as segmentation goes. We just have to sit and make extra comments about the translation, let us know if that has to be done before the transformation so that we can prioritize it accordingly.

**Comment by schola0208** [comment]
> @kklimek1 @bcristiana @elenipamboukis I am working on the file for the next 20 minutes or so. I'll notify you when I upload it

file uploaded again!!

**Comment by RJP43** [comment]
I think you all are probably at a good point for me to transform your file into HTML. Do you all agree? Please react to this issue with 👍 if you agree. You're ready to transform if your file is free of validation errors and you've implemented all of the basic structural markup. It might help to see how the file appears in the HTML output in moving forward with correcting any of the translation segmentation markup and in trying to pin down where people and places are being marked in one div (translation/transcription) but not the other.  Let me know your thoughts @schola0208 @bcristiana @kklimek1 @elenipamboukis 💟

**Comment by RJP43** [comment]
Hi team :wave: Thank you all so much for your hard work. I am closing this issue since you all are now moving on to various HTML proofing tasks. Please see issue #44 for HTML proofing instructions. 💟
